### ISSUE TITLE
Is there a way to fire jmx_prometheus_javaagent programmatically?

### ISSUE THREAD

**Issue by ashish-singh-kr** [prometheus/jmx_exporter#525]
java -javaagent:./jmx_prometheus_javaagent-0.13.0.jar=9090:config.yaml -jar Main.jar

How can I do it from a program.

I tried these. no luck!!

            try{
            String nameOfRunningVM = ManagementFactory.getRuntimeMXBean().getName();
            String pid = nameOfRunningVM.substring(0, nameOfRunningVM.indexOf('@'));
            VirtualMachine vm = VirtualMachine.attach(pid);
            vm.loadAgent("/Users/zop2816/Downloads/jmx_prometheus_javaagent-0.13.0.jar=9090:/Users/zop2816/Downloads/example.java.helloworld-master/config.yaml", null);
            vm.detach();
        }catch (Exception e) {
            Exception o = e;
            e.printStackTrace();
        }

Please help. TIA.

**Comment by brian-brazil** [comment]
It makes more sense to ask questions like this on the [prometheus-users mailing list](https://groups.google.com/forum/#!forum/prometheus-users) rather than in a GitHub issue. On the mailing list, more people are available to potentially respond to your question, and the whole community can benefit from the answers provided.

**Comment by ashish-singh-kr** [comment]
Thank you so much. I will move his question there.
